cs_upload.py

```diff
@@ -107,6 +171,9 @@ def create_trace(self, project_id: str, name: str, key: str, size: int) -> str:
         return response["createTrace"]["trace"]["id"]
 
     def generate_report(self, trace_id: str, profile_id: str) -> str:
+        logger.info(
+            f"Generating report (trace ID: {trace_id}, profile ID: {profile_id})"
```

Comment:
Same again: How about printing the integer database IDs rather than the GraphQL opaque IDs here?